aiootp/commons/packaging.py

```diff
@@ -32,7 +32,9 @@ def make_module(name: str, *, mapping: dict) -> FrozenNamespace:
     module = t.ModuleType(name)
     module.__dict__.update(mapping)
     sys.modules[name] = module
-    return FrozenNamespace(module.__dict__)
+    remake = FrozenNamespace()
+    remake.__dict__.update(module.__dict__)
+    return remake
```

Comment:
This goes against the proper usage guidelines being formulated in the PR. Perhaps it would be better to fix the logic in the superclass' init?

Reply:
Resolved by (https://github.com/rmlibre/aiootp/pull/11/commits/89c0e81ee9c3edfefd30e962b829657ed471367a)